src/lib/components/AffiliationsField.js

```diff
@@ -5,13 +5,12 @@
 // React-Invenio-Deposit is free software; you can redistribute it and/or modify it
 // under the terms of the MIT License; see LICENSE file for more details.
 
-import React, { Component } from 'react';
+import _get from 'lodash/get';
```

Comment:
comment: I don't really mind because I guess this is a correct ordering? And I see it actually compressed many that were in different lines so 💪 . But it seems to be autosave changes? to be careful in code not to alter the git history.

Reply:
Yes, I have auto organize import enabled. The issue is that there a few unused imports, it looks like that devs do not have ESLint well configured.
The idea is to create a ESLint module to standardize the code formatting across all repos.
Personally, I don't mind if imports are changed, but I can disabled it if you prefer. I change it only for files that I touch.